libraries/Microsoft.Bot.Builder.Azure/CosmosDBKeyEscape.cs

```diff
@@ -36,7 +36,7 @@ public static class CosmosDbKeyEscape
         /// <summary>
         /// Converts the key into a DocumentID that can be used safely with Cosmos DB.
         /// The following characters are restricted and cannot be used in the Id property: '/', '\', '?', and '#'.
-        /// More information at <see cref="Microsoft.Azure.Documents.Resource.Id"/>.
+        /// More information at <see href="https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.documents.resource.id?view=azure-dotnet"/>.
```

Comment:
For the life of me, I can't get `cref` to work and I don't know why. It would give "ambiguous reference" errors even with the whole namespace. I'm happy to leave as-is, remove the whole line, or change it to something that works. Let me know.

@mrivera-ms @JonathanFingold @johnataylor, any suggestions here?

Reply:
Where do you see this error?

Reply:
If I leave the `cref` line as shown above, it shows the "ambiguous reference" error when I update the `Microsoft.Azure.Cosmos` package to the latest. The error is shown both in linting and when I try to build.

Reply:
Can we just remove the cref's and href's altogether?  and just have the name in the string?

"Microsoft.Azure.Documents.Resource.Id"?

Reply:
To keep things simple, I recommend against using hrefs for this. From what Eric's told me, this might be an artifact of the refactoring process. You could remove the problem \<see> tags for now (in favor of code style). And then in a separate PR, after the refactoring is complete, try to add them back in.